### ISSUE TITLE
Monitoring beacons with region equal to null  does not work.

### ISSUE THREAD

**Issue by divye11** [MacKentoch/react-native-beacons-manager#78]
### Version
1.0.7

### Platform
iOS & Android

### OS version

iOS 10 & android 7.0

### Steps to reproduce
1.  install library
2.  use startMonitoringForRegion(null) 

### Expected behavior
Should scan for any beacon in region.

### Actual behavior
Does not monitor any beacons at all.

**Comment by ateufel** [comment]
the docs are not 100% clear on this, but i believe it´s a required parameter. it works for me with ANY string.

**Comment by divye11** [comment]
So if i put any valid format of string it will detect any and every beacon broadcasting that format?(for eg ibeacon format)

**Comment by ateufel** [comment]
just try it ;) - it worked for me with some estimote beacons.

**Comment by divye11** [comment]
We are talking about the identifier parameter of region or UUID here? Could you show me an example where you put any random uuid string for reference and could catch all beacon signals of that format(for eg ibeacon) because I had to explicitly mention all uuid and region details

**Comment by ateufel** [comment]
we are only talking about the parameter regionId, the uuids are optional. if you do not add it, you will get all beacons in range. i just used the "Simple example" code from the Readme.md for android, nothing else.

**Comment by divye11** [comment]
Yes identifier is compulsory but can be anything but in the example you just pointed out it does use a uuid: parameter in defining region. I was interested in scanning any and every beacon in my proximity without specifying uuid for every beacon, Iam using an iOS device for testing

**Comment by ateufel** [comment]
just ignore the uuid parameter then. you were talking about the region parameter, not the uuid...?

**Comment by divye11** [comment]
@ateufel  No i was talking about the uuid parameter of region
